handler.go

```diff
@@ -131,6 +133,8 @@ func NewRouter(handler *Handler) *mux.Router {
 	router.HandleFunc("/index/{index}/input-definition/{input-definition}", handler.handleDeleteInputDefinition).Methods("DELETE")
 	router.HandleFunc("/index/{index}/query", handler.handlePostQuery).Methods("POST")
 	router.HandleFunc("/index/{index}/time-quantum", handler.handlePatchIndexTimeQuantum).Methods("PATCH")
+	router.HandleFunc("/index/{index}/attr", handler.handleGetIndexAttributes).Methods("GET")
```

Comment:
for consistency, should these method names be `handleGetIndexAttr`? or is that confusing to use `Attr` for both a single col/row Attribute set as well as the entire attribute data set for an index?